src/warc2zim/content_rewriting/__init__.py

```diff
@@ -0,0 +1,2 @@
+from warc2zim.content_rewriting.html import HtmlRewriter
+from warc2zim.content_rewriting.css import CSSRewriter
```

Comment:
For style and consistency with our other rewriters, `CssRewriter` (`ss` in lower case) is probably more appropriate than `CSSRewriter`